### ISSUE TITLE
Consider loosening 'requirements' for naming certain container elements

### ISSUE THREAD

**Issue by scottaohara** [w3c/aria#2180]
Follow on to #427, radiogroup, table, grid, dialog, etc. - these are currently 'required' to be named from author, but this requirement is commonly confused as being a wcag requirement (these roles being unnamed won't actually always be a wcag failure) - and there can be instances where naming these containers can simply cause information redundancy.  

This issue is being filed to talk with the larger working group about changing the author musts to author shoulds for these sorts of roles.

**Comment by scottaohara** [comment]
from the wg call today: general agreement ARIA should not be stricter in its requirements as compared to other specs (HTML/WCAG).  end goal here would be to remove these requirements, but replace with author SHOULD statements to indicate that for most cases authors should provide names, but allow for situations where names may not be absolutely necessary

**Comment by spectranaut** [comment]
Discussed in the ARIA working group, general approval for this change: https://www.w3.org/2024/05/23-aria-minutes.html#t06
